journal/updated/02-chapter-1.adoc

```diff
@@ -128,27 +128,27 @@ In unfeigned love to you and my dear and tender children do I now write;
 and although it is so ordered that we are separated one from another outwardly,
 yet I am near to you in spirit.
 As it is in obedience to the Lord that I am where I am,
-I sincerely desire that I may continue in obedience to his pure and holy will,
-although it be a very great trial, and at times comes near almost as my life.
-Yet, blessed be the Lord, he has been pleased so to favour me with his living presence,
-and to support me with his divine power,
-that I am made willing to be what he would have me to be; and, my dear husband,
-as I desire for myself, so for you, that we both be wholly resigned to his will;
-and I believe, while we abide in his fear, all things will work together for our good.
+I sincerely desire that I may continue in obedience to His pure and holy will,
+although it be a very great trial, and at times comes near almost to my life.
+Yet blessed be the Lord, for He has been pleased so to favour me with His living presence,
+and to support me with His divine power,
+that I am made willing to be what He would have me to be. And, my dear husband,
+as I desire for myself, so for you, that we both be wholly resigned to His will;
+and I believe, while we abide in His fear, all things will work together for our good.
```

Comment:
This is kinda weird in the original. Unless I'm missing something, I think she means to say something like: **And as I desire for myself, so I desire for my dear husband, that we both**